amr-wind/utilities/sampling/Sampling.cpp

```diff
@@ -65,6 +73,26 @@ void Sampling::initialize()
         ioutils::add_var_names(m_var_names, fld.name(), fld.num_comp());
     }
 
+    // Process field information
+    m_nicomp = 0;
+    for (const auto& fname : int_field_names) {
+        if (!repo.field_exists(fname)) {
+            amrex::Print()
+                << "WARNING: Sampling: Non-existent int_field requested: "
+                << fname
+                << ". This is a mistake or the requested  int_field is a "
```

Comment:
extra space before int?

Reply:
sharp eye! thanks for catching that, fixed